man/src/oxide-config.1.md

```diff
@@ -0,0 +1,265 @@
+% OXIDE-CONFIG(1) oxide-config 0.1.0
+% Philipp Kalinowski
+% February 2023
+
+# NAME
+
+oxide-config - config
+
+# DESCRIPTION
+
+Define the behaviour of oxidewm. You can change style, layout, keybindings and more.
+The config file is written using YAML syntax.
+If the home config file is not existing, default values will be used but commands like `exec` and `exec_always` will not be working.
+
+# FILES
+
+When starting, Oxide has two paths it searches for config files.
+
+**~/.config/oxide/config.yml**
+: Home config file
+
+**/etc/oxide/config.yml**
+: System config file
+
+# KEYBINDING
+
+## KEYS
+
+A keybinding has to consist of at least one or more MODIFIERS and exactly one normal key such as 't' for example.
+
+## MODIFIER
+
+**M**
+: Meta key
+
+**A**
+: ALT key
+
+**C**
+: CONTROL key
+
+**S**
+: SHIFT key
+
+# COMMANDS
+
+Commands consist of a command and optional arguments.
+
+## COMMAND
+
+Move **args** [MOVEMENT]
+: Move window
+
+Focus **args** [MOVEMENT]
+: Move Focus
+
+Quit
+: Quit the window manager
+
+Kill
+: Kill the currently focused window
+
+Restart
+: Reloads the config and restarts components
+
+Layout **args** [LAYOUT]
+: Change the current layout
+
+GoToWorkspace **args** [WORKSPACE_ARGS]
+: Change the current workspace
+
+MoveToWorkspace **args** [WORKSPACE_ARGS]
+: Move the used window to a different workspace
+
+MoveToWorkspaceAndFollow **args** [WORKSPACE_ARGS]
+: Move the focused window to and select a different workspace
+
+Exec **args** <COMMAND>
+: Execute a given command
+
+Fullscreen
+: Toggle fullscreen mode for the focused window
+
+## ARGS
+
+Command arguments are necessary for the movement, the layout or to controll workspaces
+
+## MOVEMENT
+
+Left
+: Moves to the left
+
+right
+: Moves to the right
+
+## LAYOUT
+
+VerticalStriped
+: windows vertically next to each other
+
+HorizontalStriped
+: windows horizontally underneath each other
+
+None
+: if no argument is provided, the next layout is chosen
+
+## WORKSPACE_ARGS
+
+Next
+: next initialized workspace with a higher index than the current workspace
+
+Previous
+: next initialized workspace with a lower index than the current workspace
+
+Next_free
+: next available workspace with a higher index than the current workspace which is not initialized
+
+Index
+: workspace with the given index
+
+## ITERATIONS
```

Comment:
make this a level one header and give a little further information on the use case please, its not really understandable what this does at the moment